sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslateContent.Serialization.cs

```diff
@@ -56,14 +54,11 @@ internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
                         writer.WriteNullValue();
                         continue;
                     }
-#if NET6_0_OR_GREATER
```

Comment:
Since this code path would not have been previously triggered, I'd suggest that we remove it to preserve existing behavior.   Then we can capture the change we made in an issue and let the Azure ML team decide whether or not to add the special case back in with updated logic.

Reply:
See: https://github.com/Azure/azure-sdk-for-net/issues/47131